### DISCUSSION TITLE
Welcome to LaTerminal Discussions!

### DISCUSSION THREAD

**Discussion by migueldeicaza** [xibbon/LaTerminal#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other community members. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by calebhailey** [comment]
Greetings 👋 congrats on the launch! 🚀 Cool idea to use GitHub Discussions for community. Will you also add an RSS feed to [your blog](https://xibbon.com/blog) for those of us who want to follow along that way?

**Reply by JosephHill** [reply]
Done: https://xibbon.com/feed.xml

Thanks!
